src/main/java/com/whalespottingjava/models/validation/annotations/LatitudeConstraint.java

```diff
@@ -0,0 +1,16 @@
+package com.whalespottingjava.models.validation.annotations;
+
+import com.whalespottingjava.models.validation.validators.LatitudeValidator;
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.*;
```

Comment:
it is me being extremely picky as your code look good to me :)
I have been told it is bad practice to import everything with * in line 6
feel free to discard my comment